client-src/elements/chromedash-feature-page.ts

```diff
@@ -346,7 +346,6 @@ export class ChromedashFeaturePage extends LitElement {
   }
 
   refetch() {
-    this.loading = true;
```

Comment:
Does this change any existing behavior?

Reply:
Yes, it makes the page less jumpy after reviewer votes.  It used to be that the whole page would show skeletons briefly, and then new info would show up in the form of a gate changing color and the message at the top of the gate column changing.  Now, those change happen in a much smoother way.  

I suppose what is lost is the visual feedback in situation where the refetch takes a long time.  If we find that that happens a lot or users mention it, we could add a more subtle loading indicator.